php/wordcountRevB.php

```diff
@@ -15,7 +15,7 @@ function cmp($pairA, $pairB)
 
 $array = array();
 while (false !== ($line = fgets($stdin))) {
-    $words = preg_split('/\s+/', $line);
+    $words = explode(' ', $line);
```

Comment:
I know; it's not the same; but as you can see; a few lines down there's !empty(); so this can be easily handled; but it depends on the liking.

I don't like using regular expressions where performance is a huge factor; but that's just me. :)